tests/select_for_update/tests.py

```diff
@@ -322,3 +395,99 @@ def test_ordered_select_for_update(self):
         with transaction.atomic():
             qs = Person.objects.filter(id__in=Person.objects.order_by('-id').select_for_update())
             self.assertIn('ORDER BY', str(qs.query))
+
+    def run_select_for_update_with_related_objects_scenario(self, of):
```

Comment:
I agree, this is testing the DB, not Django. I removed them.

BTW, there are similar tests which already exist in the file for NOWAIT, that's what I cargo-culted in the first place.